### ISSUE TITLE
Advancing science by designing for surprise

### ISSUE THREAD

**Issue by delalamo** [delalamo/SKM#329]
**Authors:** James Evans; Casey Petroff; Gary King

**Venue:** Science
**Date:** 2026-07-16

**Relevance:** 0.252625

## Identifiers

- DOI: [10.1126/science.aej4257](https://doi.org/10.1126/science.aej4257)
- Source: crossref `10.1126/science.aej4257`

## Abstract

Consider a thought experiment. Two graduate students have written identical dissertations: the same experiments, the same data, the same robust findings. One is written as a standard, cautious text that flatters the subfield whose intuitions drove the design and to whom the findings will feel familiar. The other reframes the study for a different, but equally relevant, audience—one holding different suppositions and therefore open to being surprised into shifting its position. The latter approach is more valuable for good reasons: the scientific community and the world will learn more. The job of a scientist is to explain and predict the world, and a vital part of this objective is to deliver knowledge in a way that updates the understandings and expectations of the largest audience. However, scientists have long treated “framing” as a secondary art form. Students are advised to do rigorous empirical work, sprinkled with creative brilliance for which there is neither definition nor strict formula. But what if audiences’ expectations and understandings could be measured and used to guide what scientists choose to study, what conclusions they draw from the evidence, and how they frame findings for distribution? What if the framing and choice of audience are adjustable parameters of a new, broader theory of inference? The question grows particularly urgent as literature becomes flooded with machine-produced papers written by no one, for no one.

## BibTeX

The collision-checked citation key is `evans2026`; the entry below is ready to copy.

```bibtex
@article{evans2026,
  title = {Advancing science by designing for surprise},
  author = {James Evans and Casey Petroff and Gary King},
  year = {2026},
  journal = {Science},
  abstract = {Consider a thought experiment. Two graduate students have written identical dissertations: the same experiments, the same data, the same robust findings. One is written as a standard, cautious text that flatters the subfield whose intuitions drove the design and to whom the findings will feel familiar. The other reframes the study for a different, but equally relevant, audience—one holding different suppositions and therefore open to being surprised into shifting its position. The latter approach is more valuable for good reasons: the scientific community and the world will learn more. The job of a scientist is to explain and predict the world, and a vital part of this objective is to deliver knowledge in a way that updates the understandings and expectations of the largest audience. However, scientists have long treated “framing” as a secondary art form. Students are advised to do rigorous empirical work, sprinkled with creative brilliance for which there is neither definition nor strict formula. But what if audiences’ expectations and understandings could be measured and used to guide what scientists choose to study, what conclusions they draw from the evidence, and how they frame findings for distribution? What if the framing and choice of audience are adjustable parameters of a new, broader theory of inference? The question grows particularly urgent as literature becomes flooded with machine-produced papers written by no one, for no one.},
  doi = {10.1126/science.aej4257},
  url = {https://doi.org/10.1126/science.aej4257},
}
```

**Comment by delalamo** [comment]
#### Candidate atomic notes

- **Scientific informativeness is relative to the expectations of a relevant audience.** [@evans2026] The opening argument and the Shannon-linked paragraph on PDF pp. 1–2 define a paper's contribution by whose worldview it updates and by how much, not only by what it reports. This is a conceptual extension of information-theoretic surprise, not an empirical result; relevance, evidential validity, and uncertainty in the audience's prior are necessary qualifications that the article does not formalize. No existing note states this claim; it extends [[Uncertainty quantification]] from model predictions to community expectations without contradicting it. Related: [[Uncertainty quantification]]; [[Protein language models have an implicit understanding of the uncertainty of their predictions]]. Placement: new atomic note, `Scientific informativeness is relative to the expectations of a relevant audience.md`.

- **Abductive framing preserves an anomalous result, whereas HARKing erases the anomaly by presenting a post hoc hypothesis as expected.** [@evans2026] PDF p. 2 contrasts Peirce's abduction with HARKing and links Kerr's 1998 HARKing reference; the proposed sequence is to state the prevailing expectation, expose its violation, and collect further evidence. This is an epistemological distinction rather than a tested intervention, and post hoc “abductive” framing remains vulnerable to audience-shopping or reverse-HARKing. Related: [[Composition of negative data affects rule discovery in binary classification tasks]]; [[Uncertainty quantification]]. Placement: new atomic note, `Abductive framing preserves anomalies that HARKing erases.md`.

- **Answers endorsed more often than respondents predict can recover truths held by informed minorities better than simple majority vote.** [@evans2026] PDF pp. 2–3 invokes Prelec, Seung, and McCoy's 2017 “surprisingly popular” method; their linked Nature study explains the mechanism in Figure 2 and compares aggregation methods across tasks in Figures 1–4. The evidence supports survey aggregation under that study's information assumptions, but the article's analogy from survey respondents to scientific papers has not itself been validated. Related: [[Averaging logits from multiple sources can improve fitness prediction]]; [[Uncertainty quantification]]. Placement: new atomic note, `Unexpectedly popular answers can recover truths held by informed minorities.md`.

- **Papers combining unexpectedly distant research contents and disciplinary contexts are about five times more likely to become top-decile citation hits.** [@evans2026] PDF p. 3 links Shi and Evans (2023), whose Figure 2d reports the approximately fivefold association for the highest joint content-and-context novelty in MEDLINE; Figure 2c separates content and context novelty, and Supplementary Figures 4–6 extend the analysis to physics, patents, and top-1% “super hits.” This is a large observational association with citation impact, not evidence that surprise causes impact or that citations measure truth; the underlying study also analyzes only published work and represents papers coarsely using keywords and cited venues. Related: [[Aggregate benchmark correlations can mask weak within-category performance]]; [[Most assignments of genetic novelty due to homolog detection failure]]. Placement: new atomic note, `Unexpected combinations of research contents and contexts predict citation impact.md`.

- **Scientists cite familiar disciplinary contexts roughly 500% more intensely than distant contexts.** [@evans2026] PDF p. 1 links the same Shi and Evans analysis; its Figure 3a shows the proximity-dependent citation pattern in MEDLINE and APS, whereas patent citations do not show the same falloff. The large-corpus evidence strongly supports a citation-distance association, but the article's explanation that scientists are performing for likely reviewers is inferred rather than measured, and distant work may sometimes be irrelevant. Related: [[Aggregate benchmark correlations can mask weak within-category performance]]; [[Composition of negative data affects rule discovery in binary classification tasks]]. Placement: new atomic note, `Scientists cite familiar disciplinary contexts more intensely than distant ones.md`.

- **Language models could estimate field-specific audience priors, but that proposal has not yet been validated.** [@evans2026] PDF p. 4 proposes training models on a field's literature and calibrating their expectations against scientist surveys, citation behavior, and data, but supplies no benchmark, error model, or demonstration. A missing qualification is that such models may reproduce publication and sampling biases rather than current beliefs, and those priors may drift over time. This overlaps only conceptually with [[Protein language models have an implicit understanding of the uncertainty of their predictions]]; that sequence-model result neither supports nor contradicts using literature models to measure social expectations. Related: [[Protein language models have an implicit understanding of the uncertainty of their predictions]]; [[PLMs are biased by uneven distribution of sequence data in datasets such as UniRef and UniProt]]. Placement: add a brief caveat to [[Uncertainty quantification]], rather than create a standalone note.

#### Metadata check

Issue #329 contains a BibTeX record with citekey `evans2026` and DOI `10.1126/science.aej4257`; neither the citekey, DOI, nor title appears in `bibliography.bib`. The issue's BibTeX omits final publisher metadata available in the article: volume 393, issue 6808, and article number `eaej4257`. The complete final four-page Science “Expert Voices” article was inspected; it contains no figures, tables, numbered sections, supplement, or conventional reference list, so the evidence locations use PDF pages and the article's linked references.
